docs/source/config.json

```diff
@@ -15,7 +15,7 @@
         "Migrating to Apollo Server 4": "/migration",
         "Changelog": "https://github.com/apollographql/apollo-server/blob/main/CHANGELOG.md"
       },
-      "Web framework integrations": {
+      "Integrations": {
```

Comment:
I'd suggest changing this back to "Integrations". I think folks will know what that means, and it looks better in the sidebar.

Reply:
I'm unconvinced tbh. There are lots of things that integrations could mean other than web frameworks, like the whole plugin system. If we want to keep it shorter, what about the opposite change, eg "Web frameworks" or "HTTP frameworks"?

Reply:
I like "Web frameworks", I think that could work nicely. How does that feel to you @trevor-scheer? If you agree, I'll make another PR.

Reply:
@rkoron007 I'm good with either suggestion. I'd also throw "Server frameworks" into the ring. Web frameworks to me generally has a frontend connotation (but a google search tells me that's probably my own bias, so I'd be fine with it).